src/ci/dotnet-fuzzing-tools.ps1

```diff
@@ -25,7 +25,7 @@ popd
 
 # Build SharpFuzz and our dynamic loader harness for `libfuzzer-dotnet`.
 pushd src/agent/LibFuzzerDotnetLoader
-dotnet publish . -c Release -o $ARTIFACTS/LibFuzzerDotnetLoader --sc -r win10-x64 -p:PublishSingleFile=true
+dotnet publish . -c Release -o $ARTIFACTS/LibFuzzerDotnetLoader --sc -r win10-x64
```

Comment:
We were only setting `PublishSingleFile` to keep file counts down, and keep the third-party tools folder easier to navigate. However, the non-fuzzing tasks actually need access to the underlying DLL so we can `dotnet exec` it.